### PR TITLE
dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
started working on #130

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
I clicked on something to fix the conflict in the get_statistic file, @strengejacke could you check that it's okay (I don't know why there was a conflict in the first place)

**Comment by DominiqueMakowski** [comment]
And since we don't care about travis anymore, we might as well merge this

**Comment by strengejacke** [comment]
Looks like the file get_statistic.R was added back, you can simply remove it.

**Comment by strengejacke** [comment]
> And since we don't care about travis anymore, we might as well merge this

yoda-patience-young-padawan

File: R/n_clusters.R [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  if (!requireNamespace("NbClust", quietly = TRUE)) {
+    stop("Package 'NbClust' required for this function to work. Please install it by running `install.packages('NbClust')`.")
+  }
+
+  # Run the function and suppress output and automatic plotting
+  ff <- tempfile()
+  grDevices::png(filename = ff)
+  junk <- capture.output(n <- NbClust::NbClust(x, min.nc = 2, max.nc = 9, method = "ward.D2", index = "alllong"))
+  grDevices::dev.off()
+  unlink(ff)
```
Comment (strengejacke) [inline R/n_clusters.R]: Why do we need grDevices here?

Comment (DominiqueMakowski) [reply R/n_clusters.R]: otherwise it plots a figure automatically

Comment (strengejacke) [reply R/n_clusters.R]: grrr...

Comment (DominiqueMakowski) [reply R/n_clusters.R]: I know right

**Pull request merged**